job-simulation.rb

```diff
@@ -1,4 +1,39 @@
 require './Stack.rb'
 require './Queue.rb'
 
+class Workers
+  attr_reader :waiting, :workers
+  def initialize
+    @waiting = ten_peeps
+    @workers = six_workers
+  end
 
+  def ten_peeps
+    # starts the waiting list with 10 peoples
+    waiting = Queue.new
+    @waiting = (1..10).each do |num|
+      waiting.enqueue("#{num}")
+    end
+    return waiting
+  end
```

Comment:
I'm pretty sure that the `@waiting = ` is unnecessary on line 14. If it works, it's odd to me.